src/renderer/services/chain/fees/deposit.ts

```diff
@@ -59,65 +62,120 @@ export const reloadFees$: Rx.Observable<O.Option<LoadFeesHandler>> = selectedPoo
   RxOp.map(O.map(reloadFeesByChain))
 )
 
-// State to reload deposit fees
-const { get$: reloadDepositFees$, set: reloadDepositFees } = observableState<DepositType>('asym')
+// Trigger stream to reload deposit fees
+const { stream$: reloadDepositFees$, trigger: reloadDepositFees } = triggerStream()
```

Comment:
we create new streams in case chain or amount was changed so there is no need to handle `reloadDepositFees` with any parameters